style.css

```diff
@@ -0,0 +1,88 @@
+.clearfix::before,
+.clearfix::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+body {
+  margin: 0 auto;
+  font-size: 18px;
+  max-width: 800px;
+  min-width: 670px;
```

Comment:
In reviewing project 1, I discussed the reasons not to use the `body` in lieu of a container element -- let me know if you understand that or if I should provide more info 👍🏻